package.json

```diff
@@ -63,7 +63,7 @@
     "chalk": "^2.4.2",
     "clipboardy": "^2.0.0",
     "copy-webpack-plugin": "^6.0.3",
-    "core-js": "^2.6.5",
+    "core-js": "^3.7.0",
```

Comment:
this could be a problem if we want to test our components at IE. Our components are using version 2. If we are sure that these polyfills from v3 are not loaded at our components it should be ok.